Atomic/rpm_host_install.py

```diff
@@ -11,7 +11,10 @@ class RPMHostInstall(object):
 
     @staticmethod
     def copyfile(src, dest):
-        if os.path.islink(src):
+        if os.path.isdir(src):
+            if len(os.listdir(src)) == 0:
+                os.mkdir(dest)
```

Comment:
This makes sense with the commit message but in the future it may be worth adding a comment above a block such as this. At first glance, without commit context, it seems odd.

Reply:
yes sure.  I've added a fixup commit to add a comment here.